api/clients/google_workspace_client.py

```
@@ -7,316 +11,348 @@
+                },
+            }
+            return CollectionResult(
+                evidence_type="user_directory",
+                source_system="google_workspace",
+                resource_id="users",
+                resource_name="Google Workspace Users",
+                data=evidence_data,
+                quality=quality_score,
+                compliance_controls=["CC6.1", "CC6.2", "CC6.7"],
+                collected_at=datetime.now(timezone.utc),
+            )
```

Pylint (3.3.8)

[error] Unexpected keyword argument 'evidence_type' in constructor call

(E1123)